run_standalone_temp.sh

```diff
@@ -15,6 +16,13 @@ cargo build -p spacetimedb-standalone
 export STDB_PATH="${STDB_PATH:-$(mktemp -d)}"
 mkdir -p "$STDB_PATH/logs"
 
+function cleanup {
```

Comment:
@gefjon was this related (i.e., cleaning up the same stuff) that you were talking about earlier today as an ever growing pile of stuff in the temp dir?

Reply:
No, this appears to be unrelated.